### PR TITLE
readme instructs you to install & run  postgres, redis

### PR DESCRIPTION
**Pull request by amprokop**
Adds some further setup instructions.

Tested and local env doesn't run if i don't have PG and Redis running in Terminal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,17 @@ FOIA Requests that give veterans access to their own VA files take **way** too l
 
 ## First Time Development Setup
 
+You'll need Ruby 2.3.0, Postgres, and Redis if you don't have them.
+
+> $ rbenv install 2.3.0
+> $ brew install postgresql
+> $ brew install redis
+
+Run Postgres and Redis in separate terminal tabs.
```
Comment (shanear) [inline README.md]: If you use homebrew it takes care of starting these on start up.

you can run
`brew info postgresql`
for more info.

Comment (amprokop) [inline README.md]: neat, done.

Comment (amprokop) [inline README.md]: @shanear — think it's better if I add `brew services start postgresql` to the README instead?

**Comment by shanear** [comment]
nice!

File: README.md (hunk 1 above)
Comment (amprokop) [inline README.md]: and `brew services start redis`

**Pull request merged**